src/main/resources/templates/email/programme-created/v1.0.0.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">NHS England - <span
+    th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : _" th:remove="tag"></span
+  > - Onboarding to your Training programme</th:block>
+<h2>Content</h2>
+<th:block th:fragment="content">
+  <div th:replace="fragments/greeting/v1.0.0 :: body"></div>
+  <p>Congratulations on your appointment to the <b><span
+      th:text="${not #strings.isEmpty(programmeName)} ? ${programmeName} : 'programme'"></span></b>.</p>
+  <p>Please take some time to read this email in its entirety. It contains vital information to help you start the onboarding processes for your training programme and there are some actions that you will need to do straight away.</p>
+  <p><b>Please note:</b></p>
+  <p><ul>
+  <li>We are in process of ensuring your Training Programme Director(s) (TPDs) have your details.</li>
+  <li>also be aware NHS England is not your employer, we act on behalf of your Postgraduate Dean and manage your training programme for the duration of your training.</li>
+</ul>
+  </p>
+  <p><b>Timeline:</b></p>
+  <p>Over the next couple of months, you will receive a series of automated messages from NHS England via TSS (TIS Self Service) and email communications from colleagues within <span th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : 'your local deanery office'"></span
+  >, with some vital pieces of information.</p>
+  <p><b>Placement Confirmation</b></p>
+  <p>At 12 weeks prior to your commencement in post, you will receive a placement notification from us. Please note this does not constitute an offer of employment but is a confirmation of the placement you will be due to undertake and mirrors the information that has been provided to your employing organisation.</p>
+  <p>At this point, your employing organisation will follow the NHS Employment Check Standards, details of which can be found <a href="https://www.nhsemployers.org/topics-0/employment-standards-and-regulation">here</a>.</p>
+  <p><b>National Training Number (NTN)</b></p>
+  <p>Subject to you completing all actions on TIS Self Service, on the first day <span
+      th:text="${not #strings.isEmpty(startDate)} ? ${startDate} : _"></span> of your programme, you will receive your National Training Number (NTN). It is also vitally important you connect to your Postgraduate Dean for Revalidation and ensure you are connected to the correct Deanery as your Designated Body on GMC Online. Please do <b>not</b> do this before <span
+      th:text="${not #strings.isEmpty(startDate)} ? ${startDate} : _"></span>. It is essential you do not connect to your trust or employing body. The notice containing your NTN will explain how to do this.</p>
```

Comment:
If we don't have the start date available we'll end up with double spacing in the first instance and an awkward sentence in the second.

Reply:
For the purposes of the pilot, there will have to be a start date, otherwise it won't match the criteria - but I agree we need to manage the possibility that its missing in general.

Reply:
I have borrowed your approach for including the text `(start date missing)` if the start date is missing